src/snowflake/snowpark/dataframe.py

```diff
@@ -3738,6 +3756,10 @@ def copy_into_table(
             statement_params: Dictionary of statement level parameters to be set while executing this action.
             copy_options: The kwargs that is used to specify the ``copyOptions`` of the ``COPY INTO <table>`` command.
         """
+
+        # TODO: This should be an eval operation, not an assign only as implemented here. Rather, the AST should be
```

Comment:
Create a ticket to track this?

Reply:
taking a leaner approach as @sfc-gh-azwiegincew explained. In our diff, these TODOs will show up and then we'll need to prioritize/file tickets.